src/index.html

```diff
@@ -11,6 +11,36 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <!-- Search bar airbnb -->
+    <div class="box big" data-qa="big">
+      <img
+        src="images\search.svg"
+        alt="search loop"
+        class="imgbig"
```

Comment:
```suggestion
        class="img-big"
```
divide words in classes with `-`

Reply:
fixed